### PR TITLE
Add providers section in documentation and put current docs under it

### PR DESCRIPTION
**Pull request by mamoodi**
**Short description of the problem this fixes or functionality that this introduces. This may be used for the CHANGELOG**

Add providers section in documentation and put current docs under it

---
**Give a summary of what the PR does, explaining any non-trivial design decisions**

This adds a Providers section under the LLMs which is more accurate

Screenshot 2024-09-16 at 3 59 40 PM

---
**Link of any specific issues this addresses**

### CODE REVIEW

File: docs/sidebars.ts [hunk 1]
```diff
@@ -2,82 +2,112 @@ import type { SidebarsConfig } from "@docusaurus/plugin-content-docs";
...
+      items: [
+        {
+          type: 'doc',
+          label: 'Overview',
+          id: 'usage/llms/llms',
+        },
+        {
+          type: 'category',
+          label: 'Providers',
+          items: [
```
Comment (mamoodi) [inline docs/sidebars.ts:59]: The only change is adding this category and putting the docs under it. The rest were formatting changes regarding indentation.

**Review by tobitege — APPROVED** [review]
LGTM 📚

**Review by enyst — APPROVED** [review]

**Comment by enyst** [comment]
This got stuck at `build and publish app image` for over 3 hours. Apparently it was waiting for https://playwright.azureedge.net/builds/chromium/1084/chromium-linux-arm64.zip
I restarted the job with debug enabled.

**Comment by mamoodi** [comment]
> This got stuck at `build and publish app image` for over 3 hours. Apparently it was waiting for https://playwright.azureedge.net/builds/chromium/1084/chromium-linux-arm64.zip I restarted the job with debug enabled.

Thank you. Yes some jobs just get stuck sometimes. Haven't had time to investigate.

**Pull request merged**